packages/web/nexus/utils/email.ts

```diff
@@ -110,3 +126,69 @@ export const sendNewBadgeEmail = ({ user, badgeType }: sendNewBadgeEmailArgs) =>
     `),
   })
 }
+
+type MoosendSubscriberResponse = {
+  Code: number,
+  Error: string | null,
+  Context: {
+    ID: string
+  }
+}
+
+const subscribeUserToProductUpdates = async (user: User, db: PrismaClient) => {
+  const apiKey = process.env.MOOSEND_API_KEY
+  const listId = process.env.MOOSEND_PRODUCT_UPDATES_MAILING_LIST_ID
+
+  if (!apiKey || !listId) {
+    console.warn('No `MOOSEND_API_KEY` present, not signing user up for product updates.')
+    return
+  }
+
+  // If user already has an ID, do an update instead (only changes the URL)
+  const url = user.moosendSubscriberId
+    ? `https://api.moosend.com/v3/subscribers/${listId}/update/${user.moosendSubscriberId}.json?apikey=${apiKey}`
+    : `https://api.moosend.com/v3/subscribers/${listId}/subscribe.json?apikey=${apiKey}`
+
+  console.log(url)
```

Comment:
Is this in here for logging purposes or was it just for dev?

Reply:
just for dev, removed now.